septentrion/db.py

```diff
@@ -141,6 +156,11 @@ def get_applied_migrations(
 
 
 def is_schema_initialized(settings: configuration.Settings) -> bool:
+    with Query(settings=settings, query=query_migration_table_exists) as cur:
+        result = next(cur)
+        if result == [False]:
+            return False
```

Comment:
We already have "CREATE IF NOT EXISTS", I'm not sure what this brings to the table...?  Couldn't we just always try to create the table (if not --no-create-table) ?

Reply:
I don't understand your comment, as this function is not about table creation.